sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts

```diff
@@ -12,6 +12,7 @@ import {
 } from "@azure/core-http";
 import { KeyCredential } from "@azure/core-auth";
 import { PagedAsyncIterableIterator, PageSettings } from "@azure/core-paging";
+import "@azure/core-paging";
```

Comment:
This sticks out to me. Is there a side-effect of just importing core-tracing?

Reply:
This is for getting the async iterator polyfill for NodeJS v8.  We also use the same pattern in Storage and Keyvault libraries.